apps/external-devices/device-descriptors/Retrolink_Classic_Gamecube_Controller.json

```diff
@@ -0,0 +1,16 @@
+{
+    "deviceVendor": "Retrolink USB",
+    "deviceName": "Classic Gamecube Controller",
```

Comment:
GameCube? (does it have to be GameCube-style? or is it actually a gamecube controller?)